kernel/Dockerfile

```diff
@@ -18,7 +19,7 @@ RUN if [ $DEBUG -ne "0" ]; then \
     fi
 
 # Apply local patches
-COPY patches-4.9 /patches
+COPY patches-${KERNEL_SERIES} /patches
```

Comment:
Well I think it is a regression, that was one reason why I dah these files split, as I ran into this before testing a new version... You can do the mkdir in the Makefile I guess.